### ISSUE TITLE
Test on Android Chrome keeps waiting until Accept & Continue 

### ISSUE THREAD

**Issue by snehasis** [angular/protractor#3316]
Hi there!

While testing it on Android emulator on chrome it always asks for Accept & Continue terms of Chrome and next to Sign In as well. 

Also it will keep on the same Page until you manually click on the Accept & Continue every time.

This cause automation to click on the Android Emulator prompt and then proceed further.

It should not ask for any default terms acceptance and sign in features of Chrome.

Bug report
---

- Node Version: v4.4.5
- Protractor Version: 3
- Browser(s): Chrome
- Operating System and Version Mac OX 10.11.5
- Your protractor configuration file
   exports.config = {
  seleniumAddress: 'http://localhost:4723/wd/hub',//Remove when browser

  specs: ['stores/tests/e2e/**/*-spec.js'],
  useAllAngular2AppRoots: true,
  allScriptsTimeout: 110000,
  getPageTimeout: 100000,
  framework: 'jasmine2',
  jasmineNodeOpts: {
      isVerbose: false,
      showColors: true,
      includeStackTrace: false,
      defaultTimeoutInterval: 400000
  },
  // multiCapabilities: [{
  //     browserName: 'chrome'
  // }],
  // // Reference: https://github.com/appium/sample-code/blob/master/sample-code/examples/node/helpers/caps.js

  //ANDROID RUNNING SAMPLE
  capabilities: {
    browserName: 'chrome',
    'appium-version': '1.0',
    platformName: 'Android',
    platformVersion: 'N',
    deviceName: 'Default',
  },

  //IOS RUNNING SAMPLE
  // capabilities: {
  //   browserName: 'safari',
  //   'appium-version': '1.0',
  //   platformName: 'iOS',
  //   platformVersion: '9.3',
  //   deviceName: 'iPhone 5s',
  // },

  baseUrl: 'http://10.80.214.97:8080',
  onPrepare: function () {
    var wd = require('wd'),
      protractor = require('protractor'),
      wdBridge = require('wd-bridge')(protractor, wd);
    wdBridge.initFromProtractor(exports.config);
  }
};

- A relevant example test : protractor protractor.conf.js
- Output from running the test : 
[18:28:58] I/hosted - Using the selenium server at http://localhost:4723/wd/hub
[18:28:58] I/launcher - Running 1 instances of WebDriver
Started
- Steps to reproduce the bug:
Run a newly created Android Emulator N with Chrome com.android.chrome-41.0.2272.96-APK4Fun.com.apk
- The URL you are running your tests against (if relevant)
Local URL

Thanks
Snehasis Mohapatra

**Comment by juliemr** [comment]
@sjelin can you comment?

**Comment by sjelin** [comment]
Not really, I'll looking into it though.  One thing that jumps out at me is that `N` isn't a current version.  Otherwise, this sounds like it's probably an appium bug but I should look into it

**Comment by snehasis** [comment]
@sjelin , 
Even if I change the platform version to anything else, it still do the same thing.

**Comment by sjelin** [comment]
See https://github.com/angular/webdriver-manager/blob/master/docs/mobile.md#welcome-to-chome-ui.  We'll probably be unable to solve this.  But the good news is that in my experience your tests still runs, you just can't see the results.
